tailnet/tunnel.go

```diff
@@ -1,9 +1,16 @@
 package tailnet
 
-import "github.com/google/uuid"
+import (
+	"net/netip"
+
+	"github.com/google/uuid"
+)
+
+var legacyWorkspaceAgentIP = netip.MustParseAddr("fd7a:115c:a1e0:49d6:b259:b7ac:b1b2:48f4")
 
 type TunnelAuth interface {
 	Authorize(dst uuid.UUID) bool
+	AuthorizeIP(src uuid.UUID, ip netip.Prefix) bool
```

Comment:
In fact, maybe we should refactor this to authorize the `*proto.CoordinateRequest` instead of having multiple methods --- will make it easier to do other kinds of authorization in future

Reply:
yeah, i actually like that a lot better